### ISSUE TITLE
VPAT: Describe Dataset

### ISSUE THREAD

**Issue by dlowenberg** [CDL-Dryad/dryad-product-roadmap#1324]
- [x] Add visible help text, linked as aria-labelledby, aria-describedby or another method.
- [x] Clean up SR message output if possible
- [x] Add the correct ("Usage Notes") title. Ideally, simplify SR output (see issue 4).
- [x] Attach missing form field labels.
- [x] Diagnose and address multiple form field label issue.
- [x] Add visible labels to Related Works Type and Identifier fields
- [x] Change "remove" <a> elements to <button>

Rich Text Editor wireframe: https://3r06p1.axshare.com/#id=o849e2&p=ckedit-boxes&c=1
